packages/world/src/systems/MineSystem.sol

```diff
@@ -103,7 +95,7 @@ library MineLib {
     }
 
     ObjectTypeId ore = ores[oreIndex];
-    MinedOreCount._set(ore, mined[oreIndex] + 1);
+    MinedOreCount._set(ore, max[oreIndex] - remaining[oreIndex] + 1);
```

Comment:
why is this not `MinedOreCount._get(ore) + 1`?

Reply:
didn't want to do a storage read

Reply:
ahhh right I see